src/pages/Hwisung/Login/Login.js

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+// import Btn from './Components/Btn/Btn';
+// import IdPw from './Components/IdPw/IdPw';
+import { withRouter } from 'react-router-dom';
+import './Login.scss';
+
+class LoginHwisung extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      InputId: '',
+      InputPw: '',
+      BtnOnOff: 'LoginBtn',
+    };
+  }
+  goToMain = () => {
+    this.props.history.push('/Main'); //컴포넌트가 아니라 경로
+  };
+
+  handleIdInput = event => {
+    this.setState({
+      InputId: `${event.target.value}`,
+    });
+  };
+
+  handlePwInput = event => {
+    this.setState({
+      InputPw: `${event.target.value}`,
+    });
+  };
+
+  InvalidCheck = () => {
+    this.state.InputId.includes('@') && this.state.InputPw.length > 4
+      ? this.setState({ BtnOnOff: 'LoginBtnOn' })
+      : this.setState({ BtnOnOff: 'LoginBtn' });
```

Comment:
삼항 연산자 쓸 때 공통되는 명령어를 가장 앞으로 보낼 수 있어요! 🙂

```suggestion
    const { InputId, InputPw } = this.state;
    this.setState({
      BtnOnOff : 
      InputId.includes('@') && this.state.InputPw.length > 4 ?  
      'LoginBtnOn' : 'LoginBtn'
    })
```

Reply:
오오  감사합니다